meson.build

```diff
@@ -186,6 +186,28 @@ if not get_option('snpe-support').disabled()
   endif
 endif
 
+# tensorrt
+tensorrt_dep = dependency('', required: false)
+if not get_option('tensorrt-support').disabled()
+  cuda_vers = [
+    '11.0',
+    '10.2',
+  ]
```

Comment:
Do you need to restrict CUDA versions?

Reply:
I'm not sure.. but as described in https://github.com/NVIDIA/TensorRT, it seems that cuda-11.0 and cuda-10.2 are recommended.

Reply:
1. Such info can be added as a comment.
2. Need some warning messages if CUDA is installed but the version is not supported (or not recommended), then.

Reply:
Got it, updated.